.prettierignore

```diff
@@ -1,2 +1,4 @@
 **/docs/
 **/.devcontainer/devcontainer.json
+**/static/css/vizro-bootstrap.min.css
+**/static/css/vizro-bootstrap.min.map
```

Comment:
@huong-li-nguyen double checking, should this be 

```/vizro-bootstrap.min.map``` or ```/vizro-bootstrap.min.css.map```

Reply:
I think `.min.css.map` is the common syntax: https://stackoverflow.com/questions/21504611/what-are-the-map-files-used-for-in-bootstrap-3-x